### ISSUE TITLE
Meeting request problem

### ISSUE THREAD

**Issue by awickware** [bombadil/mod_proxy_msrpc#3]
Thanks so much for the email reply.  With your help I was able to get it running and it seems to be working for the most part.  One issue I am having is with meeting requests.  When creating a meeting request it all appears to work fine but the recipient never gets the message.  There are also some repeated errors in the apache log:

[Fri Jul 26 17:48:49.250344 2013] [proxy_msrpc:error] [pid 28857:tid 140027577169664] [client 71.81.44.135:41948] RPC_OUT_DATA: proxy_msrpc_read_server_response() returned unexpected status code 50                           2 from (null) (outlook.eplweb.com)
[Fri Jul 26 17:48:49.250788 2013] [proxy_msrpc:error] [pid 28731:tid 140027692558080] [client 71.81.44.135:39669] RPC_IN_DATA: Failed to sync Outlook Session ab2fcf18-36f7-f5d2-6726-94134edd8485: 2
[Fri Jul 26 17:48:49.250841 2013] [proxy_msrpc:error] [pid 28731:tid 140027692558080] [client 71.81.44.135:39669] RPC_IN_DATA: The registered Outlook Session ab2fcf18-36f7-f5d2-6726-94134edd8485 is                            in unexpected state 'BROKEN'

Perhaps my config is missing something?

Regards,

Aaron

**Comment by awickware** [comment]
We commented out OutlookAnywhereSOCacheExpire and the errors almost stopped.  We still get a few but nearly at many as before.  We also were able to send meeting requests after making this change.

**Comment by bombadil** [comment]
Where did you comment out OutlookAnyhwereSOCacheExpire? What exactly did you do?

Please provide full logs of the time of errors with debug logging for the mod_proxy_msrpc module enabled:
`LogLevel proxy_msrpc:trace2`

**Comment by awickware** [comment]
Initially we had the following in our httpd.conf:

OutlookAnywhereSOCache shmcb
OutlookAnywhereSOCacheExpire 5000000

We then commented out the OutlookAnywhereSOCacheExpire 500000 line, as the error seemed to indicate it may have been too short causing sessions to timeout.  Afterward everthing seems to be working properly.

Log:
http://pastebin.com/xCupLz34

**Comment by bombadil** [comment]
There are several, possibly independent issues to take care of here. First one is the message

```
RPC_OUT_DATA: proxy_msrpc_read_server_response() returned unexpected status code 502 from (null) (outlook.eplweb.com)
```

which indicates that the Exchange server is unhappy about the RPC_OUT_DATA request forwarded by Apache. From the data and logs I currently have, it is hard to see a reason for this. So, let's keep this issue in mind, but postpone it for now.

The other issue far further down in the log is that your Apache seems to crash after handling a MSRPC request. There are several lines with this message:

```
[Thu Aug 01 19:32:45.287496 2013] [core:notice] [pid 1566:tid 140496095729408] AH00052: child pid 5068 exit signal Segmentation fault (11)
```

What exact version (Git commit hash) did you use when you observed these crashes? Does this also happen with the most current version available in Git (i.e. commit c0145b16)? Is there any chance that we can get a backtrace of the coredumps?

**Comment by awickware** [comment]
We are currently using c0145b1.  For now all seems to working well despite the errors.  One thing we did notice is that the core errors seem to stop when we disable LogLevel proxy_msrpc:trace2.

**Comment by bombadil** [comment]
If you don't observe any issues anymore when using commit c0145b1, your issue was caused by the bad initial fix for issue #2, which got reverted in the mentioned commit. As the revert seem to have fixed the issue, I'll mark it at closed.
